src/app/story/directives/basic.component.ts

```diff
@@ -48,6 +48,17 @@ import { StoryModel, TabService } from '../../shared';
       tiptext="More is better!" >
         <div class="fancy-hint">A comma-separated list of tags relevant to the content of your piece.</div>
       </publish-fancy-field>
+      
+      <publish-fancy-field label="Release Date">
```

Comment:
Currently not seeing any changed/invalid indicators for this field(s).  Would it be worth calling out `date` as an additional `@Input`ed subtype in FancyFieldComponent?  Or alternately: explicitly bind changed/invalid classes to the input elements.